cherryontech/JourneyWise
# Feature/jw 44 update css

## Pull Request

### Description
I tweaked a little bit of the styling for the security notice. 

### Related Issue
*Continues work on [JW-44](https://marisolsaumi.atlassian.net/browse/JW-44?atlOrigin=eyJpIjoiZDhjMGY4NGMwMzk3NDUxMWEzMDA1YWY5MGI3ZDU2MjYiLCJwIjoiaiJ9)

### Description of Changes
* Added a new component for the photo collage at the top of the security notice. 
* When this security notice was merged with my most recent version of the homepage, some issues with layering of content occurred. I had to go in and adjust the z-index of my previous work, as well as the security notice, to ensure that there is no overlap on the modal.
* Tweaked margins and styling, added in the text from Ria. 

### Checklist
- [ ] I have updated the documentation or comments where necessary.

### Testing Instructions
* At the top of the browser, Mahlet placed a button where you can click to see the security notice pop up. 

### Screenshots (if applicable)

<img width="1388" alt="Screenshot 2024-05-04 at 1 17 37 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/b4af8ca8-197b-40c8-b20e-d5d9041ba4f6">
<img width="1407" alt="Screenshot 2024-05-04 at 1 17 46 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/ed7ffef9-a559-4ad9-9e2b-b97fba757187">
<img width="502" alt="Screenshot 2024-05-04 at 1 17 54 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/e6e7475d-b89c-4e64-88f9-85333996d44b">


### Gotchas/Learnings
* I still need to get more familiar with Tailwind. There are some flex grow/shrink issues  (especially with some of the photos and vectors) that need to be solved, but for now the homepage and security notice can function as is. 

### Additional Notes (Optional)
* N/A
